content/zh/docs/concepts/security/index.md

```diff
@@ -848,3 +848,4 @@ Istio 使用双向 TLS 将某些信息从客户端安全地传递到服务器。
 
 - 阅读[常见问题](/zh/docs/ops/common-problems/security-issues/)，
   以便在出现问题时更好地解决安全策略问题。
+  
```

Comment:
```suggestion

```

Reply:
Some problem for the new line, forget this suggestion, just remove two spaces and leave the new line here is OK.